phpstan.neon.dist

```diff
@@ -35,15 +31,13 @@ parameters:
         # Temporary exclusion during upgrade to API Platform 3
         - 'src/Sylius/Bundle/ApiBundle/**.php'
 
-        # Random errors related to Template Events (we're going to replace them with Twig Events anyway)
-        - 'src/Sylius/Bundle/UiBundle/Console/Command/DebugTemplateEventCommand.php'
-
         # Optional Winzou State Machine
         - 'src/Sylius/Abstraction/StateMachine/src/WinzouStateMachineAdapter.php'
 
     ignoreErrors:
-        - '/(Interface|Class) [a-zA-Z\\]+ specifies template type (\w+) of interface [a-zA-Z\\]+ as [a-zA-Z\\]+ (of [a-zA-Z\\]+ )?but it''s already specified as/' # turns off a weird generics check behavior, we are basing on Psalm for generics checks
-        - '/Access to an undefined property Doctrine\\Common\\Collections\\ArrayCollection/'
-        - '/Method Symfony\\Component\\HttpKernel\\CacheWarmer\\WarmableInterface\:\:warmUp\(\) invoked with 2 parameters\, 1 required\./' # Symfony compatibility
-        - '/Method Sylius\\Component\\(\w+)\\Model\\(\w+)\:\:getId\(\) has no return type specified./'
+        - identifier: missingType.generics # This need to be fixed, too many classes are using generics without PHPDoc type
+        - identifier: varTag.nativeType
+        - identifier: return.unusedType
```

Comment:
Those identified errors can be fixed in another PR but there is more than 400 issues here.